config.ts

```diff
@@ -129,6 +145,57 @@ interface HomePage {
 const homePage = z.strictObject({
   sections: z.array(homeSection),
 });
+
+// Listing Types
+export enum ListingType {
+  Developer = 'Developer',
+  Creative = 'Creative',
+  Managerial = 'Managerial',
+  Volunteer = 'Volunteer',
+}
+
+const listingTypeEnum = z.nativeEnum(ListingType);
+
+interface Requirement {
+  description: string;
+  icon_path?: string;
+}
+
+const requirement = z.strictObject({
+  description: z.string(),
+  icon_path: z.optional(z.string()),
+});
+
+export interface Listing {
+  priority?: number;
+  title: string;
+  description: string;
+  requirements: Requirement[];
+  type: (typeof ListingType)[keyof typeof ListingType];
```

Comment:
```suggestion
  type: ListingType;
```

No need todo any mapping here, it actually makes dealing with the enum logic elsewhere more annyoing lol.